internal/service/ec2/ec2_placement_group.go

```diff
@@ -52,6 +52,7 @@ func ResourcePlacementGroup() *schema.Resource {
 			},
 			"spread_level": {
 				Type:         schema.TypeString,
+				Computed:     true,
```

Comment:
Not sure if this is correct way to fix bug

Alternative I thought of was to set in read function but that seemed off to me as well